cloudera/tests/conftest.py

```diff
@@ -45,6 +45,16 @@ def instance():
     return common.INSTANCE
 
 
+@pytest.fixture
+def instance_autodiscover_include():
+    return common.INSTANCE_AUTODISCOVER_INCLUDE
```

Comment:
Since you're using a global variable to store your instance, it would be safer to deepcopy it in your fixture in case one of your tests modifies the instance. Not sure this happens in the cloudera integration, but this is pretty common in other integrations

Reply:
Done